lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -92,11 +92,13 @@ sub get_beacon_organization {
   my ($self) = @_;
 
   my $organization;
-  
-  my $description = "Ensembl creates, integrates and distributes reference datasets"
-                    . " and analysis tools that enable genomics."
-                    . " We are based at EMBL-EBI and our software"
-                    . " and data are freely available.";
+ 
+  my $description = "The European Bioinformatics Institute (EMBL-EBI)"
```

Comment:
FWIW heredocs exist for multiline text blocks. Less quoting needed!

Reply:
what you've just done is not what you want to do...